.gitmodules

```diff
@@ -1,16 +1,16 @@
 [submodule "panda"]
 	path = panda
-	url = ../../commaai/panda.git
```

Comment:
Please remove this for PR

Reply:
I had planned to, but my copy wouldn't build with the commaai references